model_templates/flask_extension_httpauth/README.md

```diff
@@ -11,7 +11,9 @@ Note: it is **not** necessary (nor recommended) to add authentication to custom
 This example is simply to demonstration the flexibility of the `custom_flask.py` hook.
 
 ## Instructions
-Create a new custom model with these files and use the Python Drop-In Environment with it
+Create a new custom model with these files and use the Python Drop-In Environment with it.
+
+**Important:** extending the web server is only available when running **without** the `--production` flag (or `PRODUCTION=1` environment variable).
```

Comment:
@yakov-g is this sufficient for noting that the new feature doesn't work in production mode? I took a look to see if we can have the actual application issue a warning if it found a `custom_flask.py` file in the model-dir **and** `--production` was active but I couldn't see a good way to do that so I'll leave that as a future exercise.

Reply:
Yes, looks good.